src/aleph/web/controllers/aggregates.py

```diff
@@ -62,17 +60,27 @@ async def address_aggregate(request: web.Request) -> web.Response:
 
         aggregates = list(
             get_aggregates_by_owner(
-                session=session, owner=address, with_info=query_params.with_info
+                session=session, owner=address,
+                with_info=query_params.with_info, keys=query_params.keys
             )
         )
 
     if not aggregates:
         raise web.HTTPNotFound(text="No aggregate found for this address")
 
+    if query_params.value_only and query_params.keys and len(query_params.keys) == 1:
+        output = {}
+        target_key = query_params.keys[0]
+        for result in aggregates:
+            output[result[0]] = result[1]
```

Comment:
Why not return it here ?

Reply:
There might be multiple but we are only interested in the target_key